NEWS

```diff
@@ -1,3 +1,9 @@
+etcd v3.2.7 (2017-09-01)
+- fix auth API that concurrent auth operations old revision error
```

Comment:
fix server-side auth so concurrent auth operations do not return old revision error